tests/TestCase/Database/QueryTest.php

```diff
@@ -3132,6 +3132,40 @@ public function testSqlCaseStatement()
     }
 
     /**
+     * Shows that bufferResults(false) will prevent client-side results buffering
+     *
+     * @return void
+     */
+    public function testUnbufferedQuery()
+    {
+        $query = new Query($this->connection);
+        $result = $query->select(['body', 'author_id'])
+            ->from('articles')
+            ->bufferResults(false)
+            ->execute();
+
+        $this->skipIf(
```

Comment:
This is the laziest way for me. I don't need to test for specific drivers, only statement features